index.html

```diff
@@ -351,8 +352,9 @@ <h2>Extensibility</h2>
 
   <ol>
     <li>Let <var>fragment</var> be the result of invoking the <a>fragment parsing algorithm</a> with
-    the new value as <var>markup</var>, and the <a>context object</a> as the
-    <var>context element</var>.
+    the new value as <var>markup</var>, and the <a data-cite="DOM#concept-documentfragment-host">host</a>
+    object as the <var>context element</var> if the <a>context object</a> is a <a>ShadowRoot</a> object or the
+    <a>context object</a> otherwise.
```

Comment:
Can you obtain the context element in a separate earlier step? And it would be nice to make it say the "context object's host" to be extra clear where you're getting the field from.